airflow_blogpost.md

```diff
@@ -0,0 +1,137 @@
+# Introduction
+
+Today, we're announcing an integration between Versatile Data Kit and Apache Airflow, a popular workload scheduler. This blog post serves to provide an overview of the relevant parts of these two projects, how they integrate, and how to set up an instance of Airflow which will trigger deployed VDK jobs.
+
+# Basic Concepts Overview
+
+Apache Airflow is "a platform to programmatically author, schedule and monitor workflows". It allows users to schedule workloads to be ran periodically, and also to build more complex workload structures by expressing them as directed acyclic graphs. The nodes of these graphs represent individual workloads, while the edges represent dependencies between workloads, meaning that a workload which is `pointed at` by another workload will only trigger if the previous workload completes successfully.
+Workloads within Airflow code are represented by operators. Operators are Python objects, which means that they are highly flexible in terms of what work is done by the actual workload. Airflow offers many different operators by default, and users can develop their own if they need more specialized behaviour, as we have done here.
+For more information on Apache Airflow, reference the official documentation here - https://airflow.apache.org/docs/apache-airflow/stable/index.html.
+
+Versatile Data Kit allows users to develop and manage workloads called Data jobs. These jobs can be entirely SQL code, entirely Python code, or a mix of both, and can be run either locally, or be deployed to a Kubernetes environment and be managed by the VDK Control Service. Deployed jobs can be scheduled to run periodically using a cron-like scheduling format.
+For more information on Versatile Data Kit reference the official documentation here -
+https://github.com/vmware/versatile-data-kit/wiki/Introduction
+
+![Versatile Data Kit Data Job Lifecycle](./blogpost_images/vdk_arch.jpeg)
+
+# Integration Overview
+
+To integrate VDK with Apache Airflow, we've created the airflow-provider-vdk Python package which features two user-facing classes:
+A job-triggering operator, named VDKOperator, which can be used to trigger a job execution using a configured connection to an instance of the VDK Control Service, given a particular job name and job team. This operator can be set to run synchronously or asynchronously: in the first case, the operator will only complete after the job execution completes and only be considered successful if the job execution itself is successful; in the second case, the operator is considered successful if the job execution is successfully triggered with no consideration for whether the job passes successfully or not;
```

Comment:
Try writing shorter sentences. Bullet points may also help

VDK provider has 2 user facing classes: VDKOperator and VDKSensor. When Data job is deployed by VDK COntrol Service, VDK Operator can be used to trigger its execution. 
To that yo uneed
* Configure Airflow connection to an instance of the VDK Control Service.
* Specify job name and team in VDKOperator. 

I am trying to illustrate the idea and trying to follow the rules https://custom-writing.org/blog/rules-of-writing. Not sure if it's better. I am not an experience blogger :) 
 
Also I am now realizing that we do not explain what the integration does . It probably seems obvious to us but not to anyone. It'd be good to give it to someone who doesn't know VDK (but knows Airflow?) to read it.